modules/viewing-metrics-for-the-single-model-serving-platform.adoc

```diff
@@ -42,6 +42,13 @@ sum(increase(vllm:request_success_total{namespace='${namespace}',model_name='${m
 sum(increase(tgi_request_success{namespace=${namespace}, pod=~'${model_name}-predictor-.*'}[${rate_interval}]))
 ----
 
+.. The following query displays the number of successful inference requests over a period of time for a model deployed with the Caikit embeddings runtime:
```

Comment:
Change Caikit embeddings runtime to Caikit standalone Runtime since we refer to it as the latter thoughout the docs